dapp/package.json

```diff
@@ -22,8 +22,8 @@
 		"upgrade:dependency": "npx yarn-upgrade-all"
 	},
 	"dependencies": {
-		"@algo-builder/algob": "^5.0.1",
-		"@algo-builder/web": "^5.0.1",
+		"@algo-builder/algob": "^6.0.0",
+		"@algo-builder/web": "^6.0.0",
```

Comment:
have you tested the dapp after upgrading the web package, to see if there are any breaking changes?

Reply:
Yup